File: leetcode/java/1450. Number of Students Doing Homework at a Given Time.java
class Solution {
    public int busyStudent(int[] startTime, int[] endTime, int queryTime) {
        int ret = 0;
        
        for(int i = 0; i < startTime.length; i++) {
            if(startTime[i] <= queryTime && queryTime <= endTime[i]) {
                ret++;
            }
        }
        
        return ret;
    }
}

File: leetcode/kotlin/1450. Number of Students Doing Homework at a Given Time.kt
class Solution {
    fun busyStudent(startTime: IntArray, endTime: IntArray, queryTime: Int): Int {
        
        var ret = 0
        
        for (index in 0 until startTime.size) {
            if(startTime[index] <= queryTime && queryTime <= endTime[index]) {
                ret++        
            }
        }
        
        return ret
    }
}
